suite-native/thp/src/hooks/useThpAutoconnectAlerts.tsx

```diff
@@ -0,0 +1,81 @@
+import { useCallback } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { removeThpAutoconnectThunk, startThpAutoconnectThunk, thpActions } from '@suite-common/thp';
+import { selectDeviceAutoconnectCredentials } from '@suite-common/wallet-core';
+import { useAlert } from '@suite-native/alerts';
+import { Translation } from '@suite-native/intl';
+import { useToast } from '@suite-native/toasts';
+
+export const useThpAutoconnectAlerts = () => {
+    const dispatch = useDispatch();
+    const { showAlert } = useAlert();
+    const { showToast } = useToast();
+
+    const autoconnectCredentials = useSelector(selectDeviceAutoconnectCredentials);
+
+    const turnOnAutoconnect = useCallback(() => {
+        dispatch(startThpAutoconnectThunk());
+    }, [dispatch]);
+
+    const turnOffAutoconnect = useCallback(async () => {
+        const result = await dispatch(
+            removeThpAutoconnectThunk({
+                credentials: autoconnectCredentials ?? [],
```

Comment:
The selector returns empty array as default, so this seems redundant.

```suggestion
                credentials: autoconnectCredentials,
```